presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -53,6 +53,7 @@
     private boolean optimizeSingleDistinct = true;
     private boolean optimizerReorderWindows = true;
     private boolean pushTableWriteThroughUnion = true;
+    private boolean pageCompressionEnabled = true;
```

Comment:
I'd start with the default being false, and once we are comfortable with this is production turn it on for everyone else.